G/GLU/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GLU"
+version = v"9.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://salsa.debian.org/xorg-team/lib/libglu.git", "d77f0cae59ce18bc7bba7b1f0c0b605224c23783")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd libglu
+./autogen.sh --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j ${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux i686 {libc=glibc},
```

Comment:
I'm thinking the platform refactor broke some printing here.

Reply:
I have the same gripe 🙂

Reply:
And I didn't think this badly breaks the wizard, that's bad